ios/keyman/Keyman/Keyman/PackageBrowserViewController.swift

```diff
@@ -11,93 +11,95 @@ import SwiftUI
 import KeymanEngine
 
 @available(iOS 11.0, *)
-class PackageBrowserViewController: UIDocumentBrowserViewController, UIDocumentBrowserViewControllerDelegate {
+class PackageBrowserViewController: UIDocumentPickerViewController, UIDocumentPickerDelegate {
+  private weak var navVC: UINavigationController?
```

Comment:
Oh, right.  Should probably remove this field; it was added for one of those "would-be subtle" attempts.  (Re: [lifecycle of the doc-picker class in relation to its delegate](https://www.iditect.com/how-to/54234901.html#comment-0-1))